.buildkite/pipeline.yml

```diff
@@ -321,13 +321,16 @@
     - cleanup() { if [ "${BUILDKITE_PULL_REQUEST}" = "false" ]; then ./ci/travis/upload_build_info.sh; fi }; trap cleanup EXIT
     - TORCH_VERSION=1.6 ./ci/travis/install-dependencies.sh
     - ./dashboard/tests/run_ui_tests.sh
+    # Enable job related tests after ray.init can accept gcs address
     - bazel test --config=ci $(./scripts/bazel_export_options)
       --test_env=RAY_gcs_grpc_based_pubsub=1
       --test_env=RAY_bootstrap_with_gcs=1
       --test_env=RAY_gcs_storage=memory -- //python/ray/dashboard/...
-      -//python/ray/dashboard:test_dashboard
       -//python/ray/dashboard:test_actor
       -//python/ray/dashboard:test_job_manager
+      -//python/ray/dashboard:test_cli_integration
+      -//python/ray/dashboard:test_http_job_server
+      -//python/ray/dashboard:test_job_submission
```

Comment:
Nice.

Reply:
@ericl  I think you misunderstood this one. We actually run all CI small/large/medium right now. 
THis three are tests that are turned off for gcs ha.

The reason is that we need `ray.init` to make it work. It's working in this PR https://github.com/ray-project/ray/pull/21129